stories/__tests__/SQFormMultiSelect.stories.test.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {render, screen, waitFor, within} from '@testing-library/react';
+import {composeStories} from '@storybook/testing-react';
+
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormMultiSelect.stories';
+
+const {SQFormMultiSelect} = composeStories(stories);
+
+const initialDropdownValue = '- -';
+
+describe('Tests for SQFormMultiSelect', () => {
+  it('should render a form with a multiselect input and dropdown options', async () => {
+    render(<SQFormMultiSelect size="auto" />);
```

Comment:
nit: Size defaults to 'auto'

Reply:
This is an issue with the story, not the component. Currently, if you don't add size to the story then it'll throw a warning.

Reply:
Thanks, I wasn't aware!

Reply:
Cody fixed it on the newest set of stories, but the fix hasn't been ported to the old ones yet.

Reply:
I _think_ I've fixed it. I believe the issue is with storybook changing numbers to strings when it passes the size prop along through `args`, which causes proptypes to throw a fit